common/cultures/culture_undead.txt

```diff
@@ -47,6 +48,64 @@ undead_group = {
 		# War laws
 		allow_looting = yes
 	}
+	forsaken = {
```

Comment:
How about include names from RPG?

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/73593649-75bb6080-4517-11ea-8e54-4f7bc4418420.png)

</details>

Male Names: Roberick, Magan, Danforth, Lansire.
Female Names: Yellen, ~~Limmy~~, Sarias, Mierelle.
Family Names: Dartfall, Blacksling,Ghoulhunter, Blastlich.

Reply:
Added names in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/368383602119b622dd5bca52b3547247020c89b7.
Dynasties in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/1a761000c28f2513d826480c4449d119b6e6ca31.